### DISCUSSION TITLE
New users are great ... dont forget the existing ones

### DISCUSSION THREAD

**Discussion by JamesMcGhee** [GitbookIO/community#411] [Feature Requests]
![image](https://github.com/GitbookIO/community/assets/31243736/d7ce0d6a-400e-4a6b-bb58-f9b3dad5c6ba)
Given the above image please show me where to click to sign-in

Not sign up I am an existing user, where is my sign-in/members/login button

Ya you don't consider the UX for the existing user and buried the Sign-In dialog under marketing materials for signing up

Please don't go that way, and don't over-optimize for marketing to new users, I get you need new users, I hope you get that if you don't retain them you will stop getting them.

PS yes I know I can click on any of the Start For Free or Signup buttons and have a sub button to sign in

What I am highlighting is that your existing users are not even an afterthought on your page, they are a tertiary function your page also provides. That is box standard with throwaway services that only care about new acquisitions and have zero customer support ... that isn't you, you are not a throwaway service please don't act like one with your main page which is first and for most an entry point to your service.

**Comment by addisonschultz** [comment]
Hey @JamesMcGhee! Thanks for pointing this out 😄 

We actually made some improvements to the navigation where if you already logged in, you'll have the option to visit the dashboard directly! But it looks like we forgot to add the option back in if you're not signed in 😅 

I've just updated this so you'll see the option to login now 😄 It'll go live shortly!

Thanks again!
